src/index.ts

```diff
@@ -1,6 +1,11 @@
-import { noDeprecatedClassnames, colorNoHex } from './rules';
+import {
+  noDeprecatedClassnames,
+  colorNoHex,
+  preferThemeColorClassnames,
+} from './rules';
 
 export const rules = {
   'no-deprecated-classnames': noDeprecatedClassnames,
   'color-no-hex': colorNoHex,
+  'no-tailwind-brand-colors': preferThemeColorClassnames,
```

Comment:
Update to `prefer-theme-color-classnames` 
```suggestion
  'prefer-theme-color-classnames': preferThemeColorClassnames,
```

Reply:
Updated here https://github.com/MetaMask/eslint-plugin-design-tokens/pull/22/commits/8593ab5e6ebe6ada1833fef3b31d0de9ca317fa7